sample-apps/communication/main.js

```diff
@@ -22,16 +23,20 @@ async function init () {
         msgElement.style.display = "none"
       }, 5000)
     }
+    // TODO(orenco): promise syntax
     veraApi.tryOpen({ activityId: 'start' })
   })
   veraApi.loaded()
+  veraApi.onCameraPose(() => {})
```

Comment:
this is a dem app, I will put a console log just to show how to use the realtime function

Reply:
Unclear

Reply:
Added, once every 300 frames